bfvmm/include/hve/arch/intel_x64/xen/xen_op.h

```diff
@@ -32,9 +32,13 @@
 #include <eapis/hve/arch/intel_x64/vmexit/wrmsr.h>
 #include <eapis/hve/arch/intel_x64/vmexit/rdmsr.h>
 #include <eapis/hve/arch/intel_x64/vmexit/io_instruction.h>
+#include <eapis/hve/arch/intel_x64/vmexit/ept_violation.h>
```

Comment:
Same thing, can we get rid of the memory stuff now?

Reply:
No we're stuck with xAPIC if we want PVH